src/Classes/Autoloader.php

```diff
@@ -3,7 +3,7 @@
 namespace MyRadio;
 
 /**
- * https://github.com/php-fig/fig-standards/blob/908fa291434a29951868b5111889ea6d0b37481a/accepted/PSR-4-autoloader-examples.md.
```

Comment:
i get that, but the line was too long :p

also, it was "hiding" some fixes they've made over the last 3 years (very few, regardless)